hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -488,6 +491,15 @@ message ReplicationManagerReportResponseProto {
   required ReplicationManagerReportProto report = 1;
 }
 
+message GetFailedDeletedBlocksTxnRequestProto {
+  optional string traceID = 1;
+  required int32 count = 2;
```

Comment:
In batch fetch support, we need a start ID and batch size, so user can do the iterate using different start ID and same batch size.  So we should add a start TxnID here.

Reply:
Updated